### PR TITLE
Scaffold page

### PR DESCRIPTION
**Pull request by mikeyrayvon**
and update font sizes

### CODE REVIEW

File: page.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $content_en = get_post_meta($post->ID, '_igv_content_en', true);
+    $contact_info = get_post_meta($post->ID, '_igv_contact_info', true);
+?>
+      <article <?php post_class('grid-row margin-top-small margin-bottom-basic'); ?> id="post-<?php the_ID(); ?>">
+        <div class="page-content grid-item item-s-12 item-m-8 active font-size-large margin-bottom-small">
```
Comment (varffs) [inline page.php:17]: this content has a 1 column gap between it and the address in the design, but probably best to restrict the `<p>` width with css as doing it with the grid looks complex for midsize screens

**Review by varffs — CHANGES_REQUESTED** [review]
everything cool except the paragraph max width for the info copy section

**Review by varffs — APPROVED** [review]

**Comment by mikeyrayvon** [comment]
isnt that a styling issue not a scaffolding issue?

**Comment by varffs** [comment]
well it could have been a grid layout issue which is in the scaffold. anyhow you edited all the font size stuff on here so scaffold/style division was fuzzy already

**Pull request merged**